### ISSUE TITLE
Add tls support for k8s ingress resource

### ISSUE THREAD

**Issue by gopenguin** [containous/traefik#2438]
### Do you want to request a *feature* or report a *bug*?
feature

### What did you do?
I've setup a k8s cluster, configured traefik, a webservice and an ingress resource with a tls cert in a secret.

### What did you expect to see?

Traefik reads the cert in the secret and uses this cert for tls termination and forwards the unencrypted requests to the backend service.

### What did you see instead?

Traefk didn't respect the tls property in the ingress and therefore didn't decrypt the https request.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.4.3
Codename:     roquefort
Go version:   go1.9.2
Built:        2017-11-14_11:14:24AM
OS/Arch:      linux/amd64

```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```
--web
--kubernetes
```

### If applicable, please paste the log output in debug mode (`--debug` switch)
N/A

**Comment by gopenguin** [comment]
This issue is related to #378 which is also about tls in kubernetes but using acme and kubernetes secrets to store the aquired certs.

**Comment by gopenguin** [comment]
This is resolved due to #2439 being merged.
